Requirements

```diff
@@ -0,0 +1,19 @@
+Functionele eisen
+1. Als medewerker wil ik in het systeem zien hoeveel producten er gekocht zijn
+2. Als medewerker wil ik in het systeem zien hoeveel producten er verkocht zijn
+3. Als medewerker wil ik in het systeem zien de prijzen van producten die gekocht en verkocht zijn
+4. Als medewerker wil ik zien hoeveel producten er en per soort op voorraad zijn
+6. Als medewerker wil ik zien waar de producten in de magazijn staan
+7. Als medewerker wil ik zien hoeveel er producten nog binnenkomen
+8. Als admin wil ik rechten kunnen zetten voor admins en medewerkers per afdeling
+9. Als admin wil ik persoonelsleden gegevens kunnen opvragen/aanpassen
+10. Als medewerker wil ik rapporten kunnen uitdraaien van de aantalen en prijzen onder punten 1,2,3,4,5 en 6
+11. Als admin wil ik rapporten kunnen uitdraaien van de persoonelsleden
+
```

Comment:
Bij al je functionele eisen vergeet je de "omdat" clausule. Dit is wel belangrijk om er achter te plakken, omdat je hiermee jou bedoeling van de eis kunt weergeven. De laatste kun je bijvoorbeeld interpreteren door een lijst van gewerkte uren uit te draaien, terwijl jij juist de verkoopcijfers van die medewerker bedoelde.